APIs/openEO/large_scale_processing.qmd

```diff
@@ -59,40 +57,40 @@ df = gpd.read_file("https://artifactory.vgt.vito.be/auxdata-public/grids/LAEA-20
 
-In our example, we used the openEO backend running on the Copernicus dataspace, which is based on Geotrellis, the execution options are documented [here](/jobconfigurations.html#customizing-batch-job-resources).
+In our case, we utilized the openEO backend hosted on the Copernicus Data Space Ecosystem, built on Geotrellis. Detailed execution options and configurations are documented [here](/jobconfigurations.html#customizing-batch-job-resources).
 
 ## Starting map production
 
-The openEO Python client provides a useful tool to run multiple processing jobs in [multiple backends](https://open-eo.github.io/openeo-python-client/cookbook/job_manager.html){target="_blank"}.
+The openEO Python client offers a powerful tool for executing multiple processing jobs across [various backends](https://open-eo.github.io/openeo-python-client/cookbook/job_manager.html){target="_blank"}.
 
-It takes a GeoJSON corresponding to your tile grid and job properties per tile and triggers a function provided by you whenever a new job needs to be created. You can configure multiple backends and set the number of parallel jobs per backend. 
+It operates by taking a GeoJSON file corresponding to the tile grid and includes job properties for each tile. Whenever a new job needs to be created, it triggers a function provided by the user. Configuration options include setting up multiple backends and specifying the number of parallel jobs per backend.
 
-Similarly, it also takes care of error handling and can be considered more resilient compared to writing a simple loop yourself.
+Moreover, the tool handles error management seamlessly, making it more resilient than manually implementing a loop.
 
-This script uses a CSV file to track your jobs, and whenever it is interrupted, it can simply resume from that CSV file, making it tolerant to failure.
+This script utilizes a CSV file to track jobs, enabling it to resume operations from the exact point of interruption. This capability enhances its tolerance to failures during processing.
 
 ![Tracking jobs by CSV](_images/trackingcsv.png)
 
 ## Errors during production
 
-It's common for some tasks to run into issues during production, which is okay if it doesn't happen too frequently. If a task fails, take a quick look at the error logs. If there's no obvious reason, a simple retry might do the trick. Sometimes, you might need to allocate more memory. 
+It's common for some tasks to encounter issues during production, which is generally acceptable if it happens occasionally. If a task fails, it is recommended to check the error logs. A simple retry might solve the problem if there's no obvious reason for the failure. Sometimes, users may need to allocate more memory.
```

Comment:
might be dangerous to write to add more memory. Maybe it is better that we write that if the job keep failing, to contact us through the forum?